### DISCUSSION TITLE
Can we create our own images for edgex_taf_common and edgex_compose to run Edegx-TAF on Riscv machine

### DISCUSSION THREAD

**Discussion by divyashree123456** [edgexfoundry/community#135] [Q&A] [answered]
Hi All,
As we have developed our own images for edgex services for riscv machine,now thinking to build a required setup for Edgex-TAF framework on RISCV machine using docker.Is there any Dockerfile available for these two Images used while running TAF.

TAF_COMMON_IMAGE=nexus3.edgexfoundry.org:10003/edgex-taf-common${USE_ARM64}:latest
COMPOSE_IMAGE=nexus3.edgexfoundry.org:10003/edgex-devops/edgex-compose${USE_ARM64}:latest

If not,please let me know how can i create these Images for Riscv.

Thanks in advance.

**Comment by cloudxxx8** [comment]
You can search in GitHub
https://github.com/edgexfoundry/edgex-taf-common
The compose image is just a dockerized docker-compose image
https://github.com/edgexfoundry/ci-build-images/tree/edgex-compose

**Reply by divyashree123456** [reply, marked as answer]
Thanks a lot @cloudxxx8 
I will go through the Links
